hnn_core/tests/test_network.py

```diff
@@ -200,50 +303,28 @@ def test_network():
             assert len(drive['events'][0][0]) == n_events  # 4
 
     # make sure the PRNGs are consistent.
-    target_times = {'evdist1': [66.30498327062551, 61.54362532343694],
+    target_times = {'evdist1': [66.30498327062551, 66.33129889343446],
```

Comment:
Note that the last randomly sampled time is dependent on the number artificial cells corresponding to the distal drive. Since I've now enforced `legacy_mode=False`, this has changed.